frontend/src/utils/APIClient.js

```diff
@@ -66,11 +66,18 @@ const faqsAPI = () => {
     return APICall(url, client)
 }
 
+const previewAPI = () => {
+    const url = PREVIEW_URL
+    const client = getClient('GET')
+    return APICall(url, client)
+}
+
```

Comment:
I reckon `preview` should be an option within the resourcesAPI, since we probably want to reuse this API in the **CSE Resources** page. Correct me if I'm wrong.

Reply:
done.